lib/lang/isEmpty.js

```diff
@@ -7,23 +7,23 @@ var hasValues = require('has-values');
  * functions, numbers, strings, nulls, objects and arrays.
  *
  * ```js
- * utils.lang.isEmpty('a');
- * //=> true
- *
- * utils.lang.isEmpty('');
+ * isEmpty('a');
  * //=> false
  *
- * utils.lang.isEmpty(1);
+ * isEmpty('');
```

Comment:
hmm.. I dont know. Just most of docs was without `utils.foo`. And yea we can add these to docs as in `template-helpers`